README.rst

```diff
@@ -0,0 +1,27 @@
+edX E-Commerce API Client |build-status| |coverage-status|
+=========================================================
```

Comment:
Minor: You should add one more `=` character here. If you upload this to PyPI, the missing `=` will cause their RST parser to fail, resulting in an un-rendered README. :tired_face: